### PR TITLE
Pointing: Check for existence of pointer reveal setting

### LINKED ISSUE #122: plug crashes with gnome-settings-daemon 3.34 (removed GSettings key)
Looks like GNOME people monkey'd around with gnome-settings-daemon 3.34 *again* (**sigh**), and removed or renamed the `locate-pointer` key from GSettings:

`
io.elementary.switchboard:7969): GLib-GIO-ERROR **: 13:45:12.561: Settings schema 'org.gnome.settings-daemon.peripherals.mouse' does not contain a key named 'locate-pointer'
`

This leads to a crash when opening the Mouse & Touchpad plug.

Any idea how to resolve this issue? Conditional compilation for g-s-d >= 3.34? I mean, ubuntu 20.04 LTS will have this change too, I guess ... unless they do compatibility downstream patches again

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #122

### CODE REVIEW

**Comment by danrabbit** [comment]
@decathorpe does this solve the issue for you? I'm not sure if the entire schema was removed or if I need to check for that specific key

File: src/Views/Pointing.vala [hunk 1]
```diff
@@ -84,19 +69,41 @@ public class MouseTouchpad.PointingView : Granite.SimpleSettingsPage {
         content_area.attach (new SettingLabel (_("Pointer size:")), 0, 0);
         content_area.attach (cursor_size_grid, 1, 0, 3);
 
-        content_area.attach (reveal_pointer_label, 0, 1);
-        content_area.attach (reveal_pointer_switch, 1, 1, 3);
-        content_area.attach (locate_pointer_help, 1, 2, 3);
+        var reveal_pointer_schema = SettingsSchemaSource.get_default ().lookup (
```
Comment (decathorpe) [inline src/Views/Pointing.vala]: I'm not sure that's enough. The schema `org.gnome.settings-daemon.peripherals.mouse` still exists, but only has the `double-click` and `drag-threshold` keys (checked with gnome-settings-daemon 3.34.2).

Comment (decathorpe) [reply src/Views/Pointing.vala]: So I guess only the `locate-pointer` key is gone.

**Comment by decathorpe** [comment]
Thanks for fixing this! I really appreciate it.

**Comment by worldofpeace** [comment]
:revolving_hearts: on my side as well @danrabbit

**Pull request merged**